pmd-dist/src/test/resources/scripts/pmdtest.bat

```diff
@@ -1,27 +1,27 @@
 @echo off
 
-:: BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+rem BSD-style license; for more info see http://pmd.sourceforge.net/license.html
 
-::
-:: Simple manual test script
-:: - code is copied from designer.bat to be tested here (so please check, it might be out of sync)
-:: - mostly the function "determine_java_version" is tested here
-:: - just run it with "designertest.bat" and look at the output
-:: - test cases are at the end of this script
-::
+rem
+rem Simple manual test script
+rem - code is copied from designer.bat to be tested here (so please check, it might be out of sync)
+rem - mostly the function "determine_java_version" is tested here
+rem - just run it with "designertest.bat" and look at the output
```

Comment:
```suggestion
rem - code is copied from pmd.bat to be tested here (so please check, it might be out of sync)
rem - mostly the function "determine_java_version" is tested here
rem - just run it with "pmdtest.bat" and look at the output
```